### ISSUE TITLE
Missing sensor Xiaomi Airfryer (careli.fryer.maf02)

### ISSUE THREAD

**Issue by fantnhu** [al-one/hass-xiaomi-miot#790]
Hi!
So far I have used another integration with AirFryer ([careli.fryer.maf02](https://home.miot-spec.com/s/careli.fryer.maf02)). I have a lot of Xiaomi devices, so I thought I'd transfer them here as well.
It works fine, but I miss the "Shake the food" sensor.
(Sometimes you have to shake the food during baking, then it beeps.)
Is there another way to make this work? Or if I can't do it, could you include it?
Thank you very much

**Comment by al-one** [comment]
There is no relevant property in the [specification](https://home.miot-spec.com/spec/careli.fryer.maf02) for this device.

**Comment by pplucky** [comment]
> There is no relevant property in the [specification](https://home.miot-spec.com/spec/careli.fryer.maf02) for this device.

As far as I could find in other custom component, this seems to correspond to SIID 3, PIID 10 (named [turn-pot](https://miot-spec.org/miot-spec-v2/spec/property?type=urn:careli-spec:property:turn-pot:0000000a)).

**Comment by fantnhu** [comment]
> There is no relevant property in the [specification](https://home.miot-spec.com/spec/careli.fryer.maf02) for this device.

I think the documentation is incomplete. I used the integration of a user named tsunglung, it was included. Moreover, many more entities and states were available. For example:
TurnPot --> You have to shake the food.
Pause2 --> When the container is not in place during shaking

I have experienced something similar with other devices: xiaomi antibacterial humidifier knows the child lock, but it is not included in the documentation or in this integration. I used this with custom integrations.

**Comment by al-one** [comment]
Try customize:
```yaml
# customize.yaml
sensor.careli_maf02_xxxx:
  select_properties: turn_pot
```

**Comment by pplucky** [comment]
> Try customize:
> 
> ```yaml
> # customize.yaml
> sensor.careli_maf02_xxxx:
>   select_properties: turn_pot
> ```

Tried with select_properties, sensor_properties and even number.properties, but none seem to work. Also tried turn-pot instead, but also no luck. Of course, I checked while the fryer was working!

Still then, if I run HA service `xiaomi_miot.get_properties` with the following data:
```
data:
  entity_id: switch.air_fryer_status
  mapping:
    - siid: 3
      piid: 10
  update_entity: true
  throw: true
```

It returns:
`[{'did': 'prop.3.10', 'siid': 3, 'piid': 10, 'code': 0, 'value': 0}]`

@al-one Any idea why?
